packages/manager/apps/container/tailwind.config.js

```diff
@@ -0,0 +1,55 @@
+import path from 'path';
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    path.join(
+      path.dirname(require.resolve('@ovhcloud/manager-components')),
+      '**/*.{js,jsx,ts,tsx}',
+    ),
+  ],
+  theme: {
```

Comment:
To avoid duplication Florian shared a solution to avoid redefinition of the whole tailwind.config.js
Could give it a try please ? https://github.com/ovh/manager/blob/af665d27810331ace237205ae49589bfd65507fa/packages/manager/apps/pci-vouchers/tailwind.config.js